galsim/base.py

```diff
@@ -58,13 +58,21 @@ class GSObject(object):
         >>> conv = galsim.Convolve([gal,psf])
 
     All of these classes are subclasses of GSObject, so you should see those docstrings for
-    more details about how to construct the various profiles.
-
-    Note that most GSObjects have some kind of size specification.  Typically, these would be
-    given in terms of arcsec, with the connection to the pixel size being given in the Pixel
-    class (0.2 arcsec/pixel in the above example).  However, you can have a more complicated
-    relationship between pixel and sky coordinates.  See BaseWCS for more details about
-    how to specify various kinds of world coordinate systems.
+    more details about how to construct the various profiles.  Here we discuss attributes and
+    methods that are common to all GSObjects.
+
+    GSObjects are always defined in sky coordinates.  So all sizes and other linear dimensions
+    should be in terms of some kind of units on the sky, arcsec for instance.  Only later (when
+    they are drawn) is the connection to pixel coordinates established via a pixel scale or WCS.
+    (See the documentation for galsim.BaseWCS for more details about how to specify various kinds
+    of world coordinate systems more complicated than a simple pixel scale.)
+
+    For instance, if you eventually draw onto an image that has a pixel scale of 0.2 arcsec/pixel,
+    then the normal thing to do would be to define your surface brightness profiles in terms of
+    arcsec and then draw with `pixel_scale=0.2`.  However, nowhere in GalSim do we enforce that sky
+    coordinates need to be in arcsec.  If you wanted, you could instead define the sizes of all
```

Comment:
Thanks - the change in language here was what I had in mind, but the other changes are good!  However, for the ChromaticAtmosphere change, since you added a bit of functionality, perhaps it would be good to use that functionality in a unit test?